morebits.css

```diff
@@ -10,58 +10,58 @@
 
 /* Define CSS variables */
 html {
-	--morebits-color-status-status: #4682B4;
-	--morebits-color-legend: #31628F;
-	--morebits-color-tooltip: #0000cd;
-	--morebits-color-info: #228B22;
-	--morebits-color-warning: #FF4500;
-	--morebits-color-titlebar-links: #3062AD;
-	--morebits-bgcolor-dialog: #F0F8FF;
-	--morebits-bgcolor-titlebar: #BCCADF;
+	--twinkle-color-status-status: #4682B4;
+	--twinkle-color-legend: #31628F;
+	--twinkle-color-tooltip: #0000cd;
+	--twinkle-color-info: #228B22;
+	--twinkle-color-warning: #FF4500;
+	--twinkle-color-titlebar-links: #3062AD;
+	--twinkle-bgcolor-dialog: #F0F8FF;
+	--twinkle-bgcolor-titlebar: #BCCADF;
```

Comment:
I thought it would be good to standardise the css variables by only having one variable for one color, but now I think having two css variables for the same color is better, depending on if twinkle or morebits is loaded. I've therefore updated it.